src/Lykke.Service.Assets/Cache/AssetsForClientCacheManager.cs

```diff
@@ -10,30 +10,26 @@
 using Lykke.Service.Assets.Core.Domain;
 using Lykke.Service.Assets.Core.Services;
 using Lykke.Service.Assets.Services.Domain;
-using Microsoft.Extensions.Caching.Distributed;
 using StackExchange.Redis;
 
 namespace Lykke.Service.Assets.Cache
 {
     [UsedImplicitly]
     public class AssetsForClientCacheManager : IAssetsForClientCacheManager
     {
-        private const string PatternClient = ":Assets:Client:";
+        private const string PatternClient = ":v2:Assets:Client:";
```

Comment:
Is old cache has TTL or it will be needed to clean manually?

Reply:
It has TTL.